app/scripts/card.js

```diff
@@ -1,12 +1,52 @@
 'use strict';
 
-const $ = require('jquery');
-require('jQuery.toggleModifier');
+const Firebase = require('firebase');
+const Vue = require('vue/dist/vue');
+const VueFire = require('vuefire');
+const Moment = require('moment');
 
-$('.js-card-queue-toggle').click( (e) => {
+require('./config.js');
+const Session = require('./session.js');
 
-    e.preventDefault();
-    const target = $(e.currentTarget);
+const database = Firebase.database();
+Vue.use(VueFire);
 
-    $(target).closest('.js-card-queue').toggleModifier('expanded');
-});
+class Card {
+
+    constructor () {
+
+        Session.check();
+        this.get();
+    }
+
+    get () {
+        let cards = new Vue({
+            el: '#js-card',
+            data: {
+                isExpanded: false
+            },
+            firebase: {
+                items: {
+                    source: database.ref('queue/v1/location/tasks'),
+                    asObject: true
+                }
+            },
+            methods: {
+                toggle: function (key, index) {
+
+                    this.isExpanded = !this.isExpanded;
+                },
+                retry: function(e) {
+
+                    console.log(e);
```

Comment:
Do you will remove this `console.log`s soon?

In the future don't commit this kind of thing. Commit this callbacks with a empty function is better.